hydroflow/examples/echoserver/client.rs

```diff
@@ -1,20 +1,36 @@
 use crate::protocol::EchoMsg;
+use crate::Opts;
 use chrono::prelude::*;
 use hydroflow::hydroflow_syntax;
 use hydroflow::util::{UdpSink, UdpStream};
 use std::net::SocketAddr;
 
-pub(crate) async fn run_client(outbound: UdpSink, inbound: UdpStream, server_addr: SocketAddr) {
-    println!("Attempting to connect to server at {:?}", server_addr);
+pub(crate) async fn run_client(outbound: UdpSink, inbound: UdpStream, opts: Opts) {
+    // server_addr is required for client
+    let server_addr = match opts.server_addr {
+        Some(addr) => {
+            println!("Connecting to server at {:?}", addr);
+            addr
+        }
+        None => panic!("Client requires a server address"),
+    };
```

Comment:
This probably ought to be
```rust
let server_addr = opts.server_addr.expect("Client requires a server address");
println!("Connecting to server at {:?}", server_addr);
```

(There might even be a fancy way to handle this logic directly in `clap`)